### PR TITLE
Add React and ReactDOM

### PR DESCRIPTION
**Pull request by jontonsoup4**
# Description

Adds React and ReactDOM to `package.json` as well as loads the main component into a div rather than straight into the body. ReactDOM is now required to use the `render` function, so the current implementation was breaking.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,11 +31,13 @@
     "jsx-loader": "^0.13.2",
     "node-libs-browser": "^0.5.0",
     "node-sass": "^2.1.1",
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2",
+    "react-hot-loader": "^1.3.0",
```
Comment (jpetitcolas) [inline package.json:36]: Are you using it?

**Review by jpetitcolas — CHANGES_REQUESTED** [review]

**Comment by jpetitcolas** [comment]
Thanks for this update! :)

File: package.json (hunk 1 above)
Comment (jontonsoup4) [reply package.json:36]: The `react-hot-loader`? It's the same as before, just in alphabetical order

Comment (jpetitcolas) [reply package.json:36]: My bad. Not fully woken up yet! :)

**Review by jpetitcolas — APPROVED** [review]

**Pull request merged**